components/library/Badge.jsx

```diff
@@ -0,0 +1,4 @@
+import styles from './Badge.module.scss'
```

Comment:
Reusable Badge component, right now used by the Last Updated text, can be used later on for more